vars/sconsBuild.groovy

```diff
@@ -17,7 +14,7 @@ def call(Map config) {
                 fi
                 scons -c
                 # scons -c is not perfect so get out the big hammer
-                rm -rf _build.external install build
+                rm -rf _build.external* install build
```

Comment:
This isn't really part of the mission of this patch.  We should avoid adding miscellaneous unrelated changes into patches so that we don't have unintended side-effects of reverting a patch for example.

Additionally, this will conflict with the [targeted fix](https://github.com/daos-stack/pipeline-lib/commit/bb98c92f768e5e25bcd5c7f742870c016ae0f0f7) for the _arch_ problem (which actually is going to probably be fixed in the reverse direction -- removing the arch support from CaRT and IOF -- which is why it's not landed, or even in a PR yet).